index.html

```diff
@@ -0,0 +1,29 @@
+<html lang="en-US">
+    <head>
+        <link rel="stylesheet" href="styles.css">
+        <link href="https://fonts.google.com/specimen/Poppins?query=poppins" rel="stylesheet">
+    </head>
+    <body>
+        <header>
+            <h1>IdeaBox</h1>
+        </header>
+        <main>
+            <form class="idea-form">
```

Comment:
Refactoring this html was so helpful and I think this is going to be way easier to add elements to from the js!